language/nb.json

```
@@ -440,8 +440,8 @@
           "default": "Avbryt"
         },
         {
-          "label": "Prosent fullført",
-          "default": "Prosent fullført"
+          "label": "Beskrivende tekst for fremdriftsindikatoren",
+          "default": "Fremdrift i notater: @markedNotes av @totalNotes notater markert som ferdige"
```

Sooorrryyyy, I disagree with myself

```suggestion
          "default": "Fremdrift i notater: @markedNotes av @totalNotes notater markert som ferdig"
```